components/SensorComponent.js

```diff
@@ -1,96 +1,92 @@
 import React, { useState, useEffect } from "react";
 import { Accelerometer } from "expo-sensors";
 import styled from "styled-components/native";
+import { View, Text, TouchableOpacity } from "react-native";
 
-// ==========================
-// = Functions
 const isShaking = (data) => {
-  // x,y,z CAN be negative, force is directional
-  // We take the absolute value and add them together
-  // This gives us the total combined force on the device
   const totalForce = Math.abs(data.x) + Math.abs(data.y) + Math.abs(data.z);
-
-  // If this force exceeds some threshold, return true, otherwise false
-  // Increase this threshold if you need your user to shake harder
   return totalForce > 1.78;
 };
 
-// ==========================
-// = Styled components
-const ShakeView = styled.View`
+const Container = styled.View`
   display: flex;
-  flex-direction: column;
+  text-align: center;
+  margin-bottom: 20px;
+  overflow-wrap:break-word;
```

Comment:
When I try to run the app on Expo Go on mobile, it throws an error about this particular style `overflow-wrap`. Maybe it's not React Native compatible, I'm not sure. I just commented it out and the app runs fine.